micronota/bfillings/tests/test_prodigal.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, micronota development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from os import getcwd
+import tempfile
+import shutil
+from unittest import TestCase, main
+from micronota.bfillings.prodigal import Prodigal, predict_genes
```

Comment:
the format we've been using in skbio and qiita, which I believe is supported by pep8 is:

1) system libs
2) external libs
3) within package libs

All separated by a single space. It isn't